.github/workflows/master.yml

```diff
@@ -62,15 +62,23 @@ jobs:
       run: |
         pip freeze
 
+    - name: Format
+      if: always()
```

Comment:
Why do you have to say that?

Reply:
So that even if one step fails, the others will be still be ran